spec/amp-video-interface.md

```diff
@@ -59,6 +59,9 @@ component's visual area.
 - Users can dismiss the docked video by flicking it out of view. Once dismissed
 by the user, docking will no longer occur.
 
+In order to use this attribute, the [`amp-video-docking`](https://github.com/ampproject/amphtml/blob/master/extensions/amp-video-docking/amp-video-docking.md)
```

Comment:
could you include the script here in `code` as well. would make for a quick copy-paste experience.